### PR TITLE
https://issues.redhat.com/browse/ACM-8227 PolicyReports update

### PR DESCRIPTION
**Pull request by dockerymick**
https://issues.redhat.com/browse/ACM-8227

### CODE REVIEW

File: release_notes/whats_new.adoc [hunk 1]
```diff
@@ -71,6 +71,15 @@ See link:../observability/observe_environments_intro.adoc#observing-environments
 //2.9:7720
 * The governance framework supports more template functions from the Sprig library. For more details, see the link:../governance/template_functions.adoc#open-source-community-functions[Supported Sprig open source functions].
 
+//2.9:6921(bug not a feature)
+* You can view the timestamp for policy violations. For more details, see link:../governance/policy_ctrl_adv_config.adoc
```
Comment (gparvin) [inline release_notes/whats_new.adoc]: The timestamp was present before so being able to view the timestamp isn't new.  It's just the timestamp now has a better meaning/value.  Can we say something like:
Governance policy entries in the Insights `PolicyReport` now use the timestamp from the policy that is in violation.
(It was using the current time)

Comment (dockerymick) [reply release_notes/whats_new.adoc]: @gparvin thanks for clarifying!

Comment (dockerymick) [inline release_notes/whats_new.adoc]: what are entries? do you mean parameters? ----> Answered in the next comment and I made updates to my suggestion
```suggestion
* As part of the Red Hat Insights integration with governance, the insights `PolicyReport` now uses the `timestamp` field from the policy that is in violation. For more details, see link:../governance/policy_ctrl_adv_config.adoc
```

Comment (gparvin) [reply release_notes/whats_new.adoc]: It's not parameters but fields in the PolicyReport API - specifically note the list of results in this sample. It's the `timestamp` field that appears in each `results` entry that was updated.

```
apiVersion: wgpolicyk8s.io/v1alpha2
kind: PolicyReport
metadata:
  creationTimestamp: "2023-10-09T21:44:22Z"
  generation: 43
  name: local-cluster-policyreport
  namespace: local-cluster
  resourceVersion: "1549603"
  uid: 188b488d-0f45-4904-8f2d-38f19850ed66
results:
- category: service_availability
  message: Prometheus metrics data will be lost when the Prometheus pod is restarted
    or recreated
  policy: PROMETHEUS_DB_VOLUME_IS_EMPTY
  properties:
    component: ccx_rules_ocp.external.rules.empty_prometheus_db_volume.report
    created_at: "2020-11-17 11:47:00"
    extra_data: '{"error_key":"PROMETHEUS_DB_VOLUME_IS_EMPTY","ocp_branch":"4.13","type":"rule"}'
    total_risk: "1"
  result: fail
  source: insights
  timestamp:
    nanos: 277431039
    seconds: 1697564336
- category: CM Configuration Management
  message: NonCompliant; violation - namespaces [nowhere] not found
  policy: policies.test-namespace
  properties:
    created_at: "2023-10-17T17:30:46Z"
    total_risk: "1"
  result: fail
  source: grc
  timestamp:
    nanos: 0
    seconds: 1697563856
scope:
  kind: cluster
  name: local-cluster
  namespace: local-cluster
summary:
  error: 0
  fail: 2
  pass: 0
  skip: 0
  warn: 0
```

Comment (gparvin) [reply release_notes/whats_new.adoc]: In the example above note that insights created the first results entry and GRC policy added the second results entry.  The update only applies to the second results entry which GRC added....

Comment (dockerymick) [reply release_notes/whats_new.adoc]: @gparvin thanks for explaining, it is clear that there is a `timestamp` field in the `category: service_availability` and `category: CM Configuration Management`. I also noticed that we do not have PolicyReport listed in the APIs doc. Should that be added to our documentation?

Comment (gparvin) [reply release_notes/whats_new.adoc]: It's not an ACM API that we would document with our other APIs.  It's a reporting API from the k8s policy workgroup.

Comment (dockerymick) [reply release_notes/whats_new.adoc]: Thanks Gus! Are you ok with my suggestion? @gparvin

Comment (gparvin) [reply release_notes/whats_new.adoc]: If it doesn't mention Insights PolicyReport in the text, it needs to move to the Observability section so it's clearer this isn't a policy timestamp but part of the insights integration.

Comment (dockerymick) [reply release_notes/whats_new.adoc]: https://github.com/stolostron/rhacm-docs/pull/5649#discussion_r1364187908

**Comment by dockerymick** [comment]
@gparvin thanks for your patience, when you have an opportunity will you re-review the PR?

**Review by gparvin — APPROVED** [review]
Thanks!

File: observability/manage_insights.adoc [hunk 2]
```diff
@@ -3,23 +3,20 @@
...
 
-View the following sections to learn how to manage and view insight `PolicyReports`:
-
-* <<search-insight-policy-report-violation,Searching for insight policy reports>>
-* <<viewing-vulnerabilities-insights,Viewing identified issues from the console>>
-
 [#search-insight-policy-report-violation]
 == Searching for insight policy reports
 
 You can search for a specific insight `PolicyReport` that has a violation, across your managed clusters.
```
Comment (oafischer) [inline observability/manage_insights.adoc]:
```suggestion
You can search for a specific insight `PolicyReport` that has a violation, across your managed clusters. Complete the following steps:
```

File: observability/manage_insights.adoc [hunk 3]
```diff
@@ -3,23 +3,20 @@
...
+. Click the _Search_ icon in the console header to navigate to the _Search_ page.
+. Enter the following query: `kind:policyreport`.
++
 *Note:* The `PolicyReport` name matches the name of the cluster. 
 
-You can also further specify your query by the insight policy violation and categories. When you select a `PolicyReport` name, you are redirected to the _Details_ page of the associated cluster. The _Insights_ sidebar is automatically displayed.
+. You can specify your query with the insight policy violation and categories. When you select a `PolicyReport` name, you are redirected to the _Details_ page of the associated cluster. The _Insights_ sidebar is automatically displayed.
 
-If the search service is disabled and you want to search for an insight, run the following command from your hub cluster:
+. If the search service is disabled and you want to search for an insight, run the following command from your hub cluster:
```
Comment (oafischer) [inline observability/manage_insights.adoc]: Might be worth adding a `+` here to push the command inline with the step 5

File: observability/manage_insights.adoc (hunk 2 above)
Comment (oafischer) [reply observability/manage_insights.adoc]: Not sure if this needs a colon since it's not a prompt, but I think it makes sense before steps

**Review by oafischer — APPROVED** [review]
2 small comments, otherwise looks great!

File: observability/manage_insights.adoc [hunk 4]
```diff
@@ -3,23 +3,20 @@
...
```
Comment (dockerymick) [inline observability/manage_insights.adoc:19]:
```suggestion
. If the search service is disabled and you want to search for an insight, run the following command from your hub cluster:
+
```

**Comment by dockerymick** [comment]
@gparvin @oafischer thanks for the reviews and approvals! Merging this PR

**Pull request merged**